__tests__/app.test.js

```diff
@@ -77,8 +77,41 @@ describe('app', () => {
             .get('/api/articles/astring')
             .expect(400)
             .then(({body}) => {
-                expect(body.msg).toBe('Invalid article id')
+                expect(body.msg).toBe('Invalid input')
             })
         })
     })
+    describe('GET /api/articles', () => {
+        it.only('status 200, return with an array of articles, each with the correct properties and ordered by created_at in descending order', () => {
+            return request(app)
+                .get('/api/articles')
+                .expect(200)
+                .then(({ body }) => {
+                    expect(body.articles).toBeInstanceOf(Array)
+                    expect(body.articles).toBeSortedBy('created_at', { descending: true })
+                    body.articles.forEach((article) => {
```

Comment:
before using a forEach please add a test for the length of the array. This way if the array you are receiving is empty the tests inside the forEach will be skipped but the length test outside will still run